vault/resource_jwt_auth_backend_test.go

```diff
@@ -30,6 +30,18 @@ func TestAccJWTAuthBackend(t *testing.T) {
 					resource.TestCheckResourceAttr("vault_jwt_auth_backend.jwt", "type", "jwt"),
```

Comment:
Could you add a check for local=false in this test case?

Reply:
✅ Added

```
=== RUN   TestAccJWTAuthBackendRole_import
--- PASS: TestAccJWTAuthBackendRole_import (0.40s)
=== RUN   TestAccJWTAuthBackendRole_basic
--- PASS: TestAccJWTAuthBackendRole_basic (0.38s)
=== RUN   TestAccJWTAuthBackendRole_update
--- PASS: TestAccJWTAuthBackendRole_update (0.59s)
=== RUN   TestAccJWTAuthBackendRole_full
--- PASS: TestAccJWTAuthBackendRole_full (0.32s)
=== RUN   TestAccJWTAuthBackendRoleOIDC_full
--- PASS: TestAccJWTAuthBackendRoleOIDC_full (1.25s)
=== RUN   TestAccJWTAuthBackendRole_fullUpdate
--- PASS: TestAccJWTAuthBackendRole_fullUpdate (0.37s)
=== RUN   TestAccJWTAuthBackendRole_fullDeprecated
--- PASS: TestAccJWTAuthBackendRole_fullDeprecated (0.48s)
=== RUN   TestAccJWTAuthBackend
--- PASS: TestAccJWTAuthBackend (1.35s)
=== RUN   TestAccJWTAuthBackend_OIDC
--- PASS: TestAccJWTAuthBackend_OIDC (0.39s)
=== RUN   TestAccJWTAuthBackend_OIDC_Provider_ConfigAzure
--- PASS: TestAccJWTAuthBackend_OIDC_Provider_ConfigAzure (0.39s)
=== RUN   TestAccJWTAuthBackend_OIDC_Provider_ConfigGSuite
    resource_jwt_auth_backend_test.go:119: JWT_SERVICE_ACCOUNT_PATH not set
--- SKIP: TestAccJWTAuthBackend_OIDC_Provider_ConfigGSuite (0.00s)
=== RUN   TestAccJWTAuthBackend_negative
--- PASS: TestAccJWTAuthBackend_negative (0.05s)
=== RUN   TestAccJWTAuthBackend_missingMandatory
--- PASS: TestAccJWTAuthBackend_missingMandatory (0.55s)
PASS
ok      github.com/hashicorp/terraform-provider-vault/vault     6.985s
```